source/pipeWith.js

```diff
@@ -0,0 +1,48 @@
+import _arity from './internal/_arity.js';
+import _curry2 from './internal/_curry2';
+import head from './head.js';
+import _reduce from './internal/_reduce.js';
+import tail from './tail.js';
+import identity from './identity.js';
+
+
+/**
+ * Performs left-to-right function composition using transforming function. The leftmost function may have
+ * any arity; the remaining functions must be unary.
+ *
+ * **Note:** The result of pipeWith is not automatically curried.
+ *
+ * @func
+ * @memberOf R
+ * @category Function
+ * @sig ((* -> *), [(o -> p), ..., (x -> y), (y -> z), ((a, b, ..., n) -> y)]) -> ((a, b, ..., n) -> z)
```

Comment:
For `pipeWith`, the variadic function should be the first item in the array instead of the last